java/kafka/deployment-ssl-auth.yaml

```diff
@@ -157,31 +157,43 @@ spec:
         - name: java-kafka-streams
           image: quay.io/strimzi-examples/java-kafka-streams:latest
           env:
-            - name: CA_CRT
+            - name: STRIMZI_SOURCE_TOPIC
+              value: my-topic
+            - name: STRIMZI_TARGET_TOPIC
+              value: my-topic-reversed
+            - name: STRIMZI_LOG_LEVEL
+              value: "INFO"
```

Comment:
Does not seem to be mentioned int he README.md. Also, isn't INFO the default? Is it really needed?